build-conf/zCEE3.properties

```diff
@@ -0,0 +1,20 @@
+# Releng properties used by language/zCEE3.groovy
+
+#
+# Comma separated list of required build properties for zCEE3.groovy
+zcee3_requiredBuildProperties=zcee3_shellEnvironment, zcee3_gradlePath, zcee3_gradle_JAVA_OPTS
+
+#
+# Shell environment used to run the gradle command
+zcee3_shellEnvironment=bash
+
+#
+# Absolute path to gradle executable
```

Comment:
Can we add a comment, that this is gradle on z/OS UNIX System Services? The script does not support a "distributed" build.